src/jinja2/filters.py

```diff
@@ -491,13 +514,13 @@ def do_join(eval_ctx, value, d="", attribute=None):
     return soft_str(d).join(map(soft_str, value))
 
 
-def do_center(value, width=80):
+def do_center(value: str, width: int = 80) -> str:
     """Centers the value in a field of a given width."""
     return str(value).center(width)
 
 
 @environmentfilter
-def do_first(environment, seq):
+def do_first(environment, seq: Sequence) -> Any:
```

Comment:
This should use a TypeVar the same of the below 2 functions, right?

Reply:
The type could even be `Iterable`